src/include/daos/drpc_modules.h

```diff
@@ -36,8 +36,15 @@
 enum drpc_module {
 	DRPC_MODULE_TEST		= 0,	/* Reserved for testing */
 	DRPC_MODULE_SECURITY_AGENT	= 1,
+	DRPC_MODULE_MGMT_SERVER		= 2,
 
 	NUM_DRPC_MODULES			/* Must be last */
 };
 
+enum drpc_mgmt_server_method {
```

Comment:
I would just call this drpc_mgmt_method if you don't intend on having any agent functionality.